fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/connection/FHIRDbProxyDatasourceConnectionStrategy.java

```diff
@@ -24,28 +24,29 @@
 
 /**
  * Hides the logic behind obtaining a JDBC {@link Connection} from the DAO code.
- * 
- * This strategy is used for configurations using the FHIR proxy datasource, 
+ *
+ * This strategy is used for configurations using the FHIR proxy datasource,
  * which supports dynamic configurations of datasources without requiring
  * the application server to restart.
- * 
+ *
  * @implNote Refactored from {@link FHIRDbDAOImpl}. Improves separation of
  *           concerns by removing connection management code from the DAO
  *           and injecting it as a strategy instead. This not only simplifies
  *           things, but also makes it easier to implement new strategies,
  *           such as using a JEE datasource directly instead of the FHIR
  *           proxy datasource used here.
  */
+@Deprecated
 public class FHIRDbProxyDatasourceConnectionStrategy extends FHIRDbConnectionStrategyBase {
```

Comment:
Correct me if I'm wrong, in our next release we can get rid of this. 
And start packing in fhir-core, fhir-config in the war.

Reply:
As we discussed, this strategy is currently the default (so we're not currently forcing users to make a config change). When we get rid of this in a future release, users will be required to change their config if they haven't already. In that future release, yes we can get rid of the shared lib jars, and package everything cleanly inside the WAR.